rocm-gdb/PKGBUILD

```diff
@@ -1,4 +1,6 @@
-# Maintainer Torsten Keßler <t dot kessler at posteo dot de>
+# Maintainer: Torsten Keßler <t dot kessler at posteo dot de>
+# Contributor: <pellegrinoprevete at gmail dot com>
+
 pkgname=rocm-gdb
 pkgver=4.2.0
 pkgrel=1
```

Comment:
Can you bump the `pkgrel` and update the `.SRCINFO` again?
See: https://wiki.archlinux.org/title/PKGBUILD#pkgrel